aiomysql/__init__.py

```diff
@@ -33,7 +38,10 @@
 from .cursors import Cursor, SSCursor, DictCursor, SSDictCursor
 from .pool import create_pool, Pool
 
-__version__ = '0.0.22'
+try:
+    __version__ = pkg_resources.get_distribution('aiomysql').version
```

Comment:
This approach is a bit outdated. I suggest using `setuptools-scm`'s `write_to` setting. We could have it generate a `_scm_version.py` file that is gitignored. And there could be a checked in `_version.py` that imports `version as __version__` from there and contains a fallback logic. This way, we could avoid having logic in the package init.

Reply:
implemented in #748